pkg/kv/kvserver/replica_rangefeed.go

```diff
@@ -590,7 +567,6 @@ func (r *Replica) disconnectRangefeedSpanWithErr(span roachpb.Span, pErr *kvpb.E
 		return
 	}
 	p.DisconnectSpanWithErr(span, pErr)
-	r.maybeDisconnectEmptyRangefeed(p)
```

Comment:
We may want to replace this with a call to `updateRangefeedFilterLocked()`. But now I'm actually really wondering about the management of this rangefeed filter as well.